src/app/components/import/ImportCsvModalContent.js

```diff
@@ -46,6 +46,7 @@ const ImportCsvModalContent = ({ file, onSetVcardContacts }) => {
     useEffect(() => {
         const parseFile = async () => {
             const preVcardsContacts = prepare(file);
+            console.log('prepared', preVcardsContacts);
```

Comment:
remove it

Reply:
oh, I fixed that but it seems I forgot to push, and then lint was not complaining for some reason...